cypress/integration/org_events_page.spec.ts

```diff
@@ -22,6 +22,20 @@ describe('/o/[orgId]/events', () => {
         cy.visit('/o/1/events');
         cy.get('[data-test="no-events-placeholder"]').should('be.visible');
     });
+
+    it('contains conditional functionality for sign-up button', () => {
```

Comment:
A better name would be something like "shows sign up button if not signed up, and undo button when signed up".